lib/graphql/schema.rb

```diff
@@ -887,6 +889,14 @@ def max_complexity(max_complexity = nil)
         end
       end
 
+      def max_validation_errors(max_validation_errors = nil)
```

Comment:
The usual pattern in this file is to call this `new_max_validation_errors`

Reply:
Hmm, I was copying this off of `max_complexity` which doesn't?

Reply:
🤷‍♂ the majority do but I guess there's inconsistencies already